### ISSUE TITLE
[Bug Report] Bad experience with autocomplete

### ISSUE THREAD

**Issue by jandieg** [vuetifyjs/vuetify#5926]
### Versions and Environment
**Vuetify:** 1.3.14
**Vue:** 2.5.21
**Browsers:** Chrome 71.0.3578.98
**OS:** Windows 10

### Steps to reproduce
As a user I want to select all names starting with "T".

So I type "t"

- The control shows all names starting with "t" ---> OK
- User selects "Trevor Hansen" ---> BAD: Here the filter is reset
- User is not able to add the next "Tucker Smith" because the filter of "t" is now gone

### Expected Behavior
- I type "t"
- The control shows all names starting with "t" 
- User selects "Trevor Hansen"
- The filter of "t" must remain (since its a multiselect)
- User is able to select "Tucker Smith"

### Actual Behavior
As described in repro steps.

### Reproduction Link
<a href="https://codepen.io/anon/pen/MZeNgo?&editable=true&editors=101" target="_blank">https://codepen.io/anon/pen/MZeNgo?&editable=true&editors=101</a>

**Comment by johnleider** [comment]
Your desired behavior is for the filter to not reset on selection?

**Comment by jandieg** [comment]
Yes. If after selecting items the menu remains displayed the filter should remain as well.
